lib/shopify_cli/theme/syncer.rb

```diff
@@ -256,7 +257,16 @@ def perform(operation)
         send(operation.method, operation.file) do |response|
           raise response if response.is_a?(StandardError)
 
-          @standard_reporter.report(operation.as_synced_message)
+          file = operation.file
+
+          if file.warnings.any?
```

Comment:
is `UnsupportedScriptWarning` the only warning that will exist on a file?

Reply:
Yes, currently, the `UnsupportedScriptWarning` is the only kind of warning that can exist in a file.